src/main/java/at/hannibal2/skyhanni/config/features/garden/pests/PestTimerConfig.kt

```diff
@@ -0,0 +1,74 @@
+package at.hannibal2.skyhanni.config.features.garden.pests
+
+import at.hannibal2.skyhanni.config.FeatureToggle
+import at.hannibal2.skyhanni.config.core.config.Position
+import com.google.gson.annotations.Expose
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorDraggableList
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorSlider
+import io.github.notenoughupdates.moulconfig.annotations.ConfigLink
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption
+
+class PestTimerConfig {
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Show the time since the last pest spawned in your garden.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    var enabled: Boolean = true
+
+    @Expose
+    @ConfigOption(name = "Only With Farming Tool", desc = "Only show the display when holding a farming tool in hand.")
+    @ConfigEditorBoolean
+    var onlyWithFarmingTool: Boolean = true
+
+    @Expose
+    @ConfigOption(name = "Only With Vacuum", desc = "Only show the time while holding a vacuum in the hand.")
+    @ConfigEditorBoolean
+    var onlyWithVacuum: Boolean = false
+
+    @Expose
+    @ConfigOption(name = "Pest Timer Text", desc = "Drag text to change the appearance of the overlay.")
+    @ConfigEditorDraggableList
+    var pestDisplay: MutableList<PestTimerTextEntry> = mutableListOf()
```

Comment:
please fill this list with default values. or disable the whole feature per default if you truly dont want to show this display per default